cmd/osbuild-composer/composer.go

```diff
@@ -307,14 +311,49 @@ func (c *Composer) Start() error {
 	if c.weldrListener != nil {
 		go func() {
 			err := c.weldr.Serve(c.weldrListener)
-			if err != nil {
+			if err != nil && err != http.ErrServerClosed {
 				panic(err)
 			}
 		}()
 	}
 
-	// wait indefinitely
-	select {}
+	sigint := make(chan os.Signal, 1)
+
+	signal.Notify(sigint, syscall.SIGTERM)
+	signal.Notify(sigint, syscall.SIGINT)
+
+	// block until interrupted
+	<-sigint
```

Comment:
Would like to see a small log message after the signal is received.

Reply:
This was in the wrong commit. Fixed.